airflow/jobs/triggerer_job_runner.py

```diff
@@ -401,7 +401,7 @@ def handle_failed_triggers(self):
             Stats.incr("triggers.failed")
 
     def emit_metrics(self):
-        Stats.gauge("triggers.running", len(self.trigger_runner.triggers))
+        Stats.gauge(f"triggers.running.{self.job.hostname}", len(self.trigger_runner.triggers))
```

Comment:
```suggestion
        Stats.gauge(f"triggers.running.{self.job.hostname}", len(self.trigger_runner.triggers))
        Stats.gauge(f"triggers.running", len(self.trigger_runner.triggers), tags={"hostname":self.job.hostname})
```

For now we are going to have to double-emit metrics which embed values like that due to OpenTelemetry's rather short max name length, but it supports tagging to make up for it.